toml/types.ts

```diff
@@ -0,0 +1,4 @@
+export type TOMLTable = {[Key in string]: TOMLValue} & {[Key in string]: TOMLValue|undefined};
```

Comment:
We'd prefer the capitalization of `TomlTable`. See https://docs.deno.com/runtime/contributing/style_guide/#naming-convention

Reply:
The second part `{[Key in string]: TOMLValue|undefined}` looks strange to me. Is it possible for key to be undefined? Is that spec compliant?